src/System.IO.Packaging/src/System/IO/Packaging/ContentType.cs

```diff
@@ -558,7 +547,7 @@ private static string ValidateToken(string token)
         /// <exception cref="ArgumentException">If the paramter value is empty</exception>
         private static string ValidateQuotedStringOrToken(string parameterValue)
         {
-            if (String.CompareOrdinal(parameterValue, String.Empty) == 0)
+            if (String.IsNullOrEmpty(parameterValue))
```

Comment:
Technically this change isn't the same as the previous code, but I think the previous code was flawed and that this is better, so let's keep it your new version.

(The difference is that if parameterValue is null, CompareOrdinal(null, String.Empty) will return -1, which isn't 0, thus we won't throw an ArgumentNullException, and then we'll try to dereference parameterValue with .Length and will end up null ref'ing.)